en/simulation/multi-vehicle-simulation.md

```diff
@@ -39,88 +38,72 @@ To test an example setup, follow the below steps,
+* **Gazebo model**: This is defined as `xacro` file in `Firmware/Tools/sitl_gazebo/models/rotors_description/urdf/<model>_base.xacro` see [here](https://github.com/PX4/sitl_gazebo/tree/02060a86652b736ca7dd945a524a8bf84eaf5a05/models/rotors_description/urdf). Currently, the model `xacro` file is assumed to end with base.xacro. This model should have an argument called  `mavlink_udp_port` which defines the UDP port on which gazebo will communicate with PX4 node. The model's `xacro` file will be used to generate an `urdf` model that contains UDP port that you select. To define the UDP port, set the `mavlink_udp_port` in the launch file for each vehicle, see [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/launch/multi_uav_mavros_sitl.launch#L37) as an example.  
   **NOTE: If you are using the same vehicle model, you don't need  a separate **`xacro`** file for each vehicle. The same **`xacro`** file is adequate.                
   **
 
-* **PX4 node**: this is the SITL PX4 app. It communicates with the simulator, Gazebo, through the same UDP port defined in the Gazebo vehicle model, i.e. `mavlink_udp_port`. To set the UDP port on the PX4 SITL app side, you need to set the `SITL_UDP_PRT` parameter in the startup file, see [here](https://github.com/PX4/Firmware/blob/master/posix-configs/SITL/init/ekf2/iris_1#L48). You also need to specify the path of the startup file, for each vehicle, in the launch file, see [here](https://github.com/PX4/Firmware/blob/master/launch/multi_uav_mavros_sitl.launch#L36). It is recommended that you set a unique mavlink ID for each vehicle in the startup file, see [here](https://github.com/PX4/Firmware/blob/master/posix-configs/SITL/init/ekf2/iris_2#L4).
+* **PX4 node**: This is the SITL PX4 app. It communicates with the simulator, Gazebo, through the same UDP port defined in the Gazebo vehicle model, i.e. `mavlink_udp_port`. To set the UDP port on the PX4 SITL app side, you need to set the `SITL_UDP_PRT` parameter in the startup file to match the `mavlink_udp_port` discussed previously, see [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/posix-configs/SITL/init/ekf2/iris_2#L46). The path of the startup file in the launch file is generated based on the `vehicle` and `ID` arguments, see [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/launch/multi_uav_mavros_sitl.launch#L36). The `MAV_SYS_ID` for each vehicle in the startup file, see [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/posix-configs/SITL/init/ekf2/iris_2#L4), should match the `ID` for that vehicle in the launch file [here](https://github.com/PX4/Firmware/blob/4d0964385b84dc91189f377aafb039d10850e5d6/launch/multi_uav_mavros_sitl.launch#L25). This will help make sure you keep the configurations consistant between the launch file and the startup file.
```

Comment:
Links to specific lines are discouraged as things can move around - I see that you've linked to a specific blobs in all the cases so they shouldn't, but is it likely that linking to what will become an older version will cause problems in future?

Reply:
If the files change down the road where the parameters shown in those lines don't exist I'd assume that the user could see that this is out of date and open an issue. If the file changes down the line, and the parameter still exists, there is no harm is using the specific version.

As it was (using master), it was linking to lines that didn't deal with the comment, which is even less helpful in a file that is full of a list of configuration parameters.

I'm not sure if using code fragments would make this more clear, or further confuse all of the parameters and how they interact. The benefit of the way it is now is that you can read it like a paragraph, and then look at the references or have the files open on a second monitor to follow along.

Reply:
I like code fragments because people can search for them and see them in the context of the current file. That said, you've thought this through, so good enough for now.  Thanks.